### ISSUE TITLE
Checkbox Group: Required/Mandatory information (*) provided for the group label as highlighted in red is not conveyed.

### ISSUE THREAD

**Issue by puru-hk** [SAP/fundamental-ngx#3823]
Description: Required/Mandatory information (*) provided for the group label as highlighted in red is not conveyed.

Expected:  Ideally, required information should be conveyed along with the UI information.

Screenshot: 
![image](https://user-images.githubusercontent.com/32538291/98773603-470bd800-240f-11eb-9f80-d16c661cb480.png)
